### DISCUSSION TITLE
Info command buffer size

### DISCUSSION THREAD

**Discussion by JuanPabloMF** [kickingvegas/casual-re-builder#22] [Q&A]
Hi,

I do love the concept of casual packages (it will help me a lot with discoverability of a few features I've been wanting to improve my knowledge).

I was testing casual re-builder and the info "i" command looks perfect (it opens actually the doc at the right place), but it feels kind of weird that it opens in the same minibuffer as the re-builder (it is so small that is is impossible to read). 

At this point I will move to another window and load the info buffer that was opened, but I was wondering if this was the desired behavior, or I am missing something: like is there some way to open quickly the info in another window, from the transient menu directly?

![image](https://github.com/user-attachments/assets/2a140d56-b717-4690-b543-fe62134ca67e)

Thanks and keep up the good work!

**Comment by kickingvegas** [comment]
@JuanPabloMF Thanks for sharing your report! I unfortunately am not able to repeat the behavior you see. Instead I see the Info buffer pop into a secondary buffer below the target buffer of RE-Builder. Perhaps you might want to inspect the variable `display-buffer-alist` and temporarily reset it to debug this. Please let me know your results. To bring this to a wider audience, I’m going to convert this issue into a discussion item to see if others are seeing this behavior as well.
